packages/eslint-plugin/.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "overrides": [
+    {
+      "files": [
+        "src/rules/best-practices/no-discouraged-words.{md,ts}",
```

Comment:
@decause you were using a specific phrase for these... should we potentially call the file `no-language-of-violence.md` or something to this effect?

Reply:
Though the name is a bit long, it conveys what the purpose of it is, so I'm not against it. 

The name we use for the file, isn't as consequential as the file itself. Since that file will contain sensitive/offensive/vulgar/hateful words, ideally we'd keep it somewhere separate that can be inherited/imported/referenced across all our projects/linters, rather than repeated in every repo.

That may require doing more than reworking the name, but lets start with what we've got.